e2e_playwright/shared/dataframe_utils.py

```diff
@@ -222,3 +222,25 @@ def select_column(
         column_width,
         has_row_marker_col=has_row_marker_col,
     )
+
+
+def get_open_cell_overlay(app: Locator) -> Locator:
+    """Get the currently open cell overlay / editor.
+
+    Parameters
+    ----------
+
+    app : Locator
+        The app.
+
+    Returns
+    -------
+    Locator
+        The open cell overlay.
+    """
+    # This is currently the best way to get the cell overlay
+    # We should eventually add a stable test ID to the cell overlay
+    # within GDG to better target it.
```

Comment:
nit: "GDG" -> glide data grid (I assume). I expect no one very familiar with the matter to know the abbreviation